tests/test_utils.py

```diff
@@ -38,3 +39,29 @@ def test_fit_gaussian():
     data = gaussian(np.arange(100), *params, norm=False)
     popt = fit_gaussian(data, A_sign=-1)
     assert np.allclose(popt, params)
+
+
+def test_reorder_axes_to_shape():
+    arr = np.zeros((512, 1024, 16), dtype=np.float32)  # E.g. burst mode JUNGFRAU data
+    res = reorder_axes_to_shape(arr, (16, 512, 1024))
+    assert res.shape == (16, 512, 1024)
+    assert res.base is arr
```

Comment:
Would you want to test if the data is actually transposed, as opposed to reshaped?